components/script/textinput.rs

```diff
@@ -625,11 +631,12 @@ impl<T: ClipboardProvider> TextInput<T> {
         self.perform_horizontal_adjustment(shift, select);
     }
 
-    pub fn adjust_horizontal_to_limit(&mut self, direction: Direction, select: Selection, update_text_cursor: bool) {
+    pub fn adjust_horizontal_to_limit(&mut self, direction: Direction,
+        select: Selection, update_text_cursor: ChangeEditPoint) {
```

Comment:
nit: I think it would be great to have the indentation like following

```rust
pub fn adjust_horizontal_to_limit(
    &mut self,
    direction: Direction,
    select: Selection,
    update_text_cursor: ChangeEditPoint
) {
```